.github/workflows/add_new_issues_to_project.yml

```diff
@@ -14,12 +14,32 @@ jobs:
         label-name: "bug"
         add-as-note: "true"
   Move_Feature_Request_To_Triage_Board:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: aareet/move-labeled-or-milestoned-issue@v2.1
+      with:
+        action-token: "${{ secrets.k8s_github_actions }}"
+        project-url: "https://github.com/orgs/terraform-providers/projects/55"
```

Comment:
I wasn't able to visit this URL. Is this the right URL?

Reply:
🙈good catch, it should be /hashicorp